plugins/inputs/logparser/README.md

```diff
@@ -108,10 +108,11 @@ CUSTOM time layouts must be within quotes and be the representation of the
 "reference time", which is `Mon Jan 2 15:04:05 -0700 MST 2006`
 See https://golang.org/pkg/time/#Parse for more details.
 
-Telegraf has many of its own
-[built-in patterns](./grok/patterns/influx-patterns),
-as well as supporting
+Telegraf has many of its own [built-in patterns](./grok/patterns/influx-patterns),
+as well as support for most of
 [logstash's builtin patterns](https://github.com/logstash-plugins/logstash-patterns-core/blob/master/patterns/grok-patterns).
+_Golang regular expressions do not support lookahead or lookbehind.
+logstash patterns that depend on these are not supported._
```

Comment:
Maybe we can have a list of the patterns linked above that are not in Telegraf?  We should probably also pin the list to a specific version.

Reply:
Is there an easy way to get the list of logstash patterns not supported by Telegraf?

Reply:
I guess not really, we could write a script that compares the pattern names
between the logstash patterns[1] with the vjeantet/grok libraries patterns[2],
but it would probably go out of date, so lets not do that.

[1] https://github.com/logstash-plugins/logstash-patterns-core/blob/master/patterns/grok-patterns
[2] https://github.com/vjeantet/grok/tree/d73e972b60935c7fec0b4ffbc904ed39ecaf7efe/patterns